apps/src/eval/eval.js

```diff
@@ -41,13 +41,9 @@ var experiments = require('../experiments');
 var ResultType = studioApp.ResultType;
 var TestResults = studioApp.TestResults;
 
-// Loading these modules extends SVGElement and puts canvg in the global
-// namespace
-var canvg = require('canvg');
+import canvg from 'canvg';
 // tests don't have svgelement
-if (typeof SVGElement !== 'undefined') {
-  require('../canvg/svg_todataurl');
-}
+import '../util/svgelement-polyfill';
```

Comment:
nope. it will only be included in the bundle once. We import all kinds of things multiple times.